app/src/main/res/layout/profile_reset_pin_activity.xml

```diff
@@ -93,6 +96,9 @@
             android:id="@+id/input_confirm_pin"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginStart="28dp"
+            android:layout_marginTop="28dp"
```

Comment:
This is `31.02dp` in mocks so it should be `32dp` in here so that it gets closest to mock.

Reply:
updated as per error text height